examples/src/createReactAgent/customSystemPrompts.ts

```diff
@@ -0,0 +1,76 @@
+/**
+ * Specify Custom System Prompt
+ *
+ * Custom system prompts allow you to define the agent's behavior, personality, and operational guidelines.
+ * This can be either a static string or a dynamic function that generates prompts based on current state.
+ *
+ * Why this is important:
+ * - Behavioral Control:
+ *   Ensures the agent consistently follows your specific guidelines and maintains
+ *   the desired tone and approach
+ * - Domain Expertise:
+ *   Allows you to inject specialized knowledge and constraints relevant to your
+ *   specific use case
+ * - Dynamic Adaptation:
+ *   Enable context-aware behavior changes based on conversation state, user type,
+ *   or external conditions
+ *
+ * **Example Scenario:**
```

Comment:
eh, i think just formatting information about the user that is passed in in context or something is fine here. i dont actually think this example is great?